src/main/java/core/basesyntax/service/impl/ParserServiceImpl.java

```diff
@@ -0,0 +1,20 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.service.ParserService;
+import java.util.Arrays;
+
+public class ParserServiceImpl implements ParserService {
+    @Override
+    public FruitTransaction parse(String lineFromFile) {
+        FruitTransaction fruitTransaction = new FruitTransaction();
+        String[] transactionData = lineFromFile.split(",");
+        fruitTransaction.setOperation(Arrays.stream(FruitTransaction.OperationType.values())
+                .filter(type -> type.getOperationType().equals(transactionData[0]))
+                .findFirst()
+                .orElse(null));
+        fruitTransaction.setFruitName(transactionData[1]);
+        fruitTransaction.setQuantity(Integer.parseInt(transactionData[2]));
+        return fruitTransaction;
+    }
```

Comment:
Are you sure we don't need validate input data?
Use constants instead of numbers when referring to array elements.